### DISCUSSION TITLE
fix order list numbering reverted in 6.1.3

### DISCUSSION THREAD

**Discussion by mtallenca** [singerdmx/flutter-quill#999] [General]
Why was this reverted?  This did fix list numbering when there was indented levels.

**Comment by singerdmx** [comment]
There is regression. See https://github.com/singerdmx/flutter-quill/issues/993

**Reply by mtallenca** [reply]
Got it.  Hopefully an update to "fix order list numbering" is quick - have a lot of users happy about that update.

**Comment by singerdmx** [comment]
@kulture-rob-snider can submit a new PR together with the reverted change and the fix.
